File: Scan_Converting_Arc_Trigonometric_Method.py

# Scan Converting Arc using Trigonometric Method
import numpy as np
import matplotlib.pyplot as plt

r = 50
theta = np.linspace(0, np.pi/2, 200)
x = r * np.cos(theta)
y = r * np.sin(theta)

plt.plot(x, y)
plt.gca().set_aspect('equal', 'box')
plt.title("Arc - Trigonometric Method")
plt.grid(True)
plt.show()

File: Scan_Converting_Arc_Trigonometric_Method.m

% Scan Converting Arc using Trigonometric Method
clc; clear; close all;
r = 50; xc = 0; yc = 0;
theta = linspace(0, pi/2, 200);
x = xc + r*cos(theta);
y = yc + r*sin(theta);
plot(x,y); axis equal; grid on;
title('Arc - Trigonometric Method');
